cmd/root.go

```diff
@@ -47,7 +47,7 @@ var rootCmd = &cobra.Command{
 }
 
 func Execute() {
-	if os.Args[0] == docker.DOCKER_CREDS_HELPER {
+	if strings.Contains(os.Args[0], docker.DOCKER_CREDS_HELPER) {
```

Comment:
should it be string.Contains or string.HasSuffix?

Reply:
I copied what we do for `git-credential-helper` below. I believe it has something to do with the `.exe` suffix on Windows.